DiSCOS/DiSCOS.md

```
@@ -270,16 +267,11 @@ ReAct, CoT, CoVe, CoN etc
     - Reductio ad absurdum
-### Reliability
-- Secure
-- Observable
-- Maintainable
-- Performant
-- Scalable
 ### Pattern Selection
 - Match Complexity to Problem
 - Consider Team Size/Maturity, Domain Volatility, Scale Requirements(current&projected)
-## Documentation
+## Documentation [IMPORTANT]
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)